english/marshmallow.md

```diff
@@ -1192,3 +1192,8 @@ Maybe this is a [dream](dream/dream.md)?
 
 Or maybe you love [pugs] (https://www.youtube.com/watch?v=3pPR7vk-e3Y)?
 
+Have you ever heard any unusual sounds 
+when you're alone? 
+They're not what you think. 
+Get to know them [here](Chapter 1/blackandWhiteShortStory.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.